### DISCUSSION TITLE
Module "hextra" is not compatible with this Hugo version??

### DISCUSSION THREAD

**Discussion by PeterSprague** [imfing/hextra#503] [Q&A]
- Trying to use Hextra template on a new site.  
- Followed the Getting Started Guide using git submodule add
- Copied the hugo.yaml and hugo.work files from example to site folder main dir  
-- Removed module: imports section from hugo.yaml
- Copied over the content dir from example to main site level

Results in a number of errors: 
- including Hugo version incompatibility 
- then failed executions of sidebar.html IsMultilingual section (line 46)  
-- Tried commenting out the section but still produces the errors.

Run:
$ hugo server --buildDrafts --disableFastRender
Errors:
Module "hextra" is not compatible with this Hugo version  <-- don't see any notes on required versions
...
Start building sites …
hugo v0.121.2+extended linux/amd64 BuildDate=2024-02-07T00:00:00+00:00 VendorInfo=Fedora:0.121.2-1.fc40

ERROR render of "home" failed: "xxxxxLab.git/themes/hextra/layouts/hextra-home.html:3:7": execute of template failed: template: hextra-home.html:3:7: executing "main" at <partial "sidebar.html" (dict "context" . "disableSidebar" true)>: error calling partial: "xxxxxLab.git/themes/hextra/layouts/partials/sidebar.html:47:11": execute of template failed: template: partials/sidebar.html:47:11: executing "partials/sidebar.html" at <hugo>: can't evaluate field IsMultilingual in type interface {}

ERROR render of "term" failed: "xxxxxLab.git/themes/hextra/layouts/_default/list.html:3:7": execute of template failed: template: _default/list.html:3:7: executing "main" at <partial "sidebar.html" (dict "context" .)>: error calling partial: "xxxxxLab.git/themes/hextra/layouts/partials/sidebar.html:47:11": execute of template failed: template: partials/sidebar.html:47:11: executing "partials/sidebar.html" at <hugo>: can't evaluate field IsMultilingual in type interface {} Built in 137 ms

Error: error building site: render: failed to render pages: render of "page" failed: "xxxxxLab.git/themes/hextra/layouts/blog/single.html:3:7": execute of template failed: template: blog/single.html:3:7: executing "main" at <partial "sidebar.html" (dict "context" . "disableSidebar" true "displayPlaceholder" true)>: error calling partial: "xxxxxLab.git/themes/hextra/layouts/partials/sidebar.html:47:11": execute of template failed: template: partials/sidebar.html:47:11: executing "partials/sidebar.html" at <hugo>: can't evaluate field IsMultilingual in type interface {}

Been around and around on this for the morning.  Any pointers or help would be appreciated.

Thanks

**Comment by PeterSprague** [comment]
Since Fedora 40 installs v0.121.2 of hugo, I installed the latest release binary v0.139.2 from github

Now doesn't throw code errors, but still indicates: 
- WARN  Module "project" is not compatible with this Hugo version: Min 0.112.0 extended;

Website main page appears with menu, but the blogs or docs just show a blank page with header even though there are both _index.html and md files present.

**Reply by imfing** [reply]
> I installed the latest release binary v0.139.2 from github

did you install the extended version? e.g. https://github.com/gohugoio/hugo/releases/download/v0.139.3/hugo_extended_0.139.3_linux-amd64.deb
